.github/quarkus-bot.yml

```diff
@@ -127,24 +127,33 @@ triage:
         - extensions/vertx-graphql/
         - integration-tests/vertx-graphql/
     - labels: [area/tracing, area/smallrye]
-      title: "(trace|tracing|telemetry)"
+      titleBody: "(trace|tracing)"
```

Comment:
I strongly suggest you don't do that. You will have a ton of false positives. I removed all the `titleBody` because it pinged people for nothing. Typically when people include the list of their extensions in the issue or when they use the word trace or tracing for another purpose.

Not doing that gives us a bit more triaging job but not that much: people usually include what's not working in the title.

Same comments for all the ``titleBody``s below.

Reply:
No worries, I switched it back